javascript/javascript1/week1/startupNameGenerator.js

```diff
@@ -0,0 +1,11 @@
+let firstWords = ["Tech", "Creative", "Bright", "Smart", "Future", "Innovator", "Dynamic", "Artistic", "Witty", "Labs", "Geometric", "Solution", "Boost", "Spark", "Empower",]
```

Comment:
`firstWords` and `secondWords` have 14 elements each (0-13 index), but you're using Math.random() * 13, which will result in an index from 0 to 12. This means the last element of both arrays is never selected.

Use `const` instead of `let`